code/__DEFINES/colors.dm

```diff
@@ -20,15 +20,19 @@
 #define COLOR_OLIVE            "#808000"
 #define COLOR_LIME             "#32CD32"
 #define COLOR_GREEN            "#008000"
+#define COLOR_VIBRANTGREEN     "#00FF00"
 #define COLOR_CYAN             "#00FFFF"
+#define COLOR_DARK_CYAN        "#00a2ff"
 #define COLOR_TEAL             "#008080"
 #define COLOR_BLUE             "#0000FF"
 #define COLOR_BLUE_LIGHT       "#33CCFF"
 #define COLOR_NAVY             "#000080"
 #define COLOR_PINK             "#FFC0CB"
 #define COLOR_MAGENTA          "#FF00FF"
+#define COLOR_LIGHT_PURPLE     "#b900f7"
 #define COLOR_PURPLE           "#800080"
 #define COLOR_ORANGE           "#FF9900"
+#define COLOR_ORANGER          "#ff7b00"
```

Comment:
```suggestion
#define COLOR_MOSTLY_PURE_ORANGE          "#ff7b00"
```

Reply:
doesn't fit on the line, going with dark orange